index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
So you don't need this file. You could just run the scale-balance.js file through command line by just doing `node scale-balance.js`